tensorflow/tools/ci_build/builds/with_the_same_user

```diff
@@ -48,6 +48,12 @@ getent passwd "${CI_BUILD_UID}" || adduser ${ADDUSER_OPTS} \
 usermod -a -G sudo "${CI_BUILD_USER}"
 echo "${CI_BUILD_USER} ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/90-nopasswd-sudo
 
+if [[ "${TF_NEED_ROCM}" -eq 1 ]]; then
+  # ROCm requires the video group in order to use the GPU for compute. If it
+  # exists on the host, add it to the container.
+  getent group video || addgroup video && adduser "${CI_BUILD_USER}" video
```

Comment:
@caisq I am quite unfamiliar with this script. Are the changes here OK?

Reply:
@parallelo This line is to fulfill permission requirements from ROCm stack specified at: https://github.com/RadeonOpenCompute/ROCm#next-set-your-permissions

Reply:
In the case of ROCm, we add the container's user to the `video` group (but only if the host user was a member of the `video` group).  This group membership is currently a requirement for ROCm.